.github/workflows/alter-primary-key-false-test.yml

```diff
@@ -0,0 +1,42 @@
+name: alter-primary-key-false-test
```

Comment:
This action test name is not clear enough. It seems like a complete set of tests. How about changing it to an actual test rather than a name of configurations.

Reply:
> This action test name is not clear enough. It seems like a complete set of tests. How about changing it to an actual test rather than a name of configurations.

Maybe later there is a test to configure alter-primary-key=false can also be placed here.